notebooks/feedback/argilla_101.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
Also, If possible , please provide an instruction to create a workspace, "e.g argilla"
Thanks @Shreyanand !

Reply:
<p>These links are usually short lived. When we move the cluster it will change, so whoever is reading the notebook should reach out for latest links.</p>